conditionally-dequeue-scripts-styles.php

```diff
@@ -3,12 +3,12 @@
  * Plugin Name: WP Simple Pay - Conditionally dequeue scripts & styles
  * Plugin URI: https://wpsimplepay.com
  * Description: Conditionally dequeue scripts & styles based on set conditions.
- * Version: 1.0
+ * Version: 1.1
  */
 
 /**
- * In this example, we'll remove all plugin scripts & styles unless on a specific page with the slug
- * "payment-page". Available WP conditional tags: https://codex.wordpress.org/Conditional_Tags
+ * In this example, we'll remove all plugin scripts & styles unless in an array of specific pages with the slugs
+ * "payment-page", "donate", "subscribe". Available WP conditional tags: https://codex.wordpress.org/Conditional_Tags
```

Comment:
What about:

> In this example, we remove all plugin scripts & styles unless the current page slug is "payment-page", "donate", or "subscribe".

Lets move the "Available WP ..." to the next line as well so the URL is easier to find.

Reply:
Makes sense, made the changes.